From 806d92cddb76a24dbc135feea0137cb95ed31490 Mon Sep 17 00:00:00 2001
From: nmlgc <nmlgc@nmlgc.net>
Date: Wed, 1 Jan 2025 03:00:52 +0100
Subject: [PATCH] Disable unconditional x86 sibling arch builds on Linux.

This breaks if 32-bit GCC libs aren't installed, such as on the x86_64
version of Flatpak's `org.freedesktop.Sdk`.
---
 linux.tup | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/linux.tup b/linux.tup
index 11ac6098..512b443b 100644
--- a/linux.tup
+++ b/linux.tup
@@ -3,7 +3,3 @@ TUP_SERVER = @(TUP_SERVER)
 ifeq (,$(TUP_SERVER))
 TUP_SERVER = fuse3
 endif
-
-ifeq (@(TUP_ARCH),x86_64)
-  X86_ARCH_SIBLINGS=y
-endif
--
2.47.1

